recipes/uncrustify/all/test_package/conanfile.py

```diff
@@ -0,0 +1,10 @@
+from conans import ConanFile, tools
+import sys
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```

Use full settings by default